cartography/intel/crxcavator/crxcavator.py

```diff
@@ -195,8 +246,34 @@ def load_extensions(extensions, session, update_tag):
     e.lastupdated = {UpdateTag}
     """
 
+    permissions_ingestion_cypher = """
+    UNWIND {Permissions} as permission
+    MERGE (e:ChromeExtensionPermission{id: permission})
+    ON CREATE SET
+    e.firstseen = timestamp()
+    SET
+    e.name = permission,
+    e.lastupdated = {UpdateTag}
+    """
+
+    extensions_permissions_cypher = """
+    UNWIND {ExtensionPermissions} as extension_permission
+    MATCH (perm:ChromeExtensionPermission{name: extension_permission.permission}),
```

Comment:
If you change the index [here](https://github.com/lyft/cartography/blob/4667376439ed96d695b633a78532560daae3eccb/cartography/data/indexes.cypher#L67) to ID, make sure that the `match` uses an indexed field.